submissions/DawoduRahmon/HTML/product-page.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
You have a index.html already that is your product page. This file is not needed. You should remove from this pull request.